charts/agent-docker/Chart.yaml

```diff
@@ -5,7 +5,7 @@ description: |
   where runs are executed in [dind](https://hub.docker.com/_/docker) sidecar container.
   Run phases are isolated into docker containers.
 type: application
-version: 0.5.40
+version: 0.6.40
```

Comment:
It’s unnecessary, we’re changing versions during the release for all charts in sync via automated workflows.